src/ontology/ro-edit.owl

```diff
@@ -5333,8 +5333,9 @@ SymmetricObjectProperty(obo:RO_0002801)
 
 # Object Property: obo:RO_0002802 (reservoir host of)
 
-AnnotationAssertion(obo:IAO_0000115 obo:RO_0002802 "A reservoir is a population, species or community (assemblage of different species in a given geographic area) in which a microorganism naturally occurs and is indefinitely maintained. Some zoonotic pathogens, particularly bacterial pathogens, may also have environmental reservoirs. Microorganisms with multiple reservoir species may be indefinitely maintained across the community of species even if they are not always present in each individual reservoir species. In the reservoir species or community, the microorganism may cause either asymptomatic infection or disease and this may vary among individuals. A pathogen may also be more genetically diverse in its reservoir host than in other taxa, in part because the pathogen is endemic to reservoir hosts.")
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0002802 "A sub-relation of host-of in which the host is a reservoir in which the hosted organism naturally occurs and is indefinitely maintained.")
```

Comment:
@wdduncan please review my edits following your suggestion to shorten the definition of reservoir host and move the quoted definition to rdfs:comment.

Reply:
Possibly "A host-of relation which attaches between a host organism and a hosted organism that naturally occurs in an indefinitely maintained reservoir that the host provides."